bin/sozo/src/commands/model.rs

```diff
@@ -109,8 +110,11 @@ hashes, called 'hash' in the following documentation.
 
         #[arg(value_name = "KEYS")]
         #[arg(value_delimiter = ',')]
-        #[arg(help = "Comma seperated values e.g., 0x12345,0x69420,...")]
-        keys: Vec<Felt>,
+        #[arg(help = "Comma separated values e.g., \
+                     0x12345,0x69420,sstr:\"hello\",sstr:\"misty\". Supported prefixes:\n  \
+                     - sstr: A cairo short string\n  \
+                     - no prefix: A cairo felt")]
```

Comment:
In the case of the keys, only `one felt long` serialized type are supported (everything that can fit in one felt).

That's great you've only put the `sstr` which is the only one supported actually.

```suggestion
        #[arg(help = "Comma separated values e.g., \
                     0x12345,0x69420,sstr:\"hello\". Supported prefixes:\n  \
                     - sstr: A cairo short string\n  \
                     - no prefix: A cairo felt")]
```

Reply:
We may add a check that only this prefix is actually used, wdyt?